src/monster.cpp

```diff
@@ -1085,17 +1085,7 @@ bool Monster::walkToSpawn()
 	return true;
 }
 
-void Monster::onWalk()
-{
-	Creature::onWalk();
-
-	if ((attackedCreature || followCreature) && isFleeing()) {
-		if (lastPathUpdate < OTSYS_TIME()) {
-			g_dispatcher.addTask(createTask([id = getID()]() { g_game.updateCreatureWalk(id); }));
-			lastPathUpdate = OTSYS_TIME() + getNumber(ConfigManager::PATHFINDING_DELAY);
-		}
-	}
-}
```

Comment:
For comparison, the code for `forceUpdatePath()`:

https://github.com/ranisalt/forgottenserver/blob/fileloader-iterators/src/creature.cpp#L145-L153